config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml

```diff
@@ -599,14 +599,15 @@ periodics:
     - args:
       - "--timeout=340"
       - "--repo=k8s.io/kubernetes=release-1.11"
+      - "--repo=k8s.io/examples=master"
```

Comment:
@krzyzacy can we checkout multiple repo?

Reply:
yes you can